tests/artist-delete.test.js

```diff
@@ -0,0 +1,57 @@
+const { expect } = require("chai");
+const request = require("supertest");
+const getDb = require("../src/services/db");
+const app = require("../src/app");
+
+describe("delete artist", () => {
+  let db;
+  let artists;
+  beforeEach(async () => {
+    db = await getDb();
+    await Promise.all([
+      db.query("INSERT INTO Artist (name, genre) VALUES(?, ?)", [
+        "Tame Impala",
+        "rock",
+      ]),
+      db.query("INSERT INTO Artist (name, genre) VALUES(?, ?)", [
+        "Kylie Minogue",
+        "pop",
+      ]),
+      db.query("INSERT INTO Artist (name, genre) VALUES(?, ?)", [
+        "Dave Brubeck",
+        "jazz",
+      ]),
+    ]);
+
+    [artists] = await db.query("SELECT * from Artist");
+  });
+
+  afterEach(async () => {
+    await db.query("DELETE FROM Artist");
+    await db.close();
+  });
+
+  describe("/artist/:artistId", () => {
+    describe("DELETE", () => {
+      it("deletes a single artist with the correct id", async () => {
+        const artist = artists[0];
+        const res = await request(app).delete(`/artist/${artist.id}`).send();
+
+        expect(res.status).to.equal(200);
```

Comment:
Would be good to move this to be with the other assertions at the end of the test